### DISCUSSION TITLE
TopNavigation href cannot be removed

### DISCUSSION THREAD

**Discussion by suvihaara** [cloudscape-design/components#807] [Q&A] [answered]
Hi, 

I wanted to check whether it is possible to remove href from the TopNavigation identity? I would like to remove the interaction from the TopNavigation title, but seems like I cannot due to href being a required property in identity.

**Comment by just-boris** [comment, marked as answer]
This is a common UI convention when the logo is clickable and contains link to the service homepage.

It is also enforced by [our usage guidelines](https://cloudscape.design/components/top-navigation/?tabId=usage)

> The service identity tells the user which service they're using. Always link it to the homepage of your particular service, so users can learn more about it.


Why do you want to render it differently?
